gateway_to_backend/protocol_buffers_files/otap_message.proto

```diff
@@ -24,11 +24,23 @@ message GetScratchpadStatusResp {
 }
 
 message UploadScratchpadReq {
+    message ChunkInfo {
+        // Id of the scratchpad this chunk belongs
+        required uint32 scratchpad_unique_id = 1;
+        // Offset of the chunk in the full scratchpad
+        // If 0, then it is the first chunk
+        required uint32 offset = 2;
```

Comment:
Is this more like a chunk sequence number? For me the offset would mean more like byte offset. Maybe "sequence_number" etc. would be better if it is just 0,1,2...

Reply:
A write block request will be converted to a call on CMeshAPI. Current API (WPC_upload_local_block_scratchpad) to write a block is based on a start address and number of bytes. Sequence must be initiated with WPC_start_local_scratchpad_update(), that defines the total expected scratchpad length, and a sequence number.
From gateway point of view, it seems easier to stick to offset = start adress and size to transfer a block.

Reply:
Good point, let's rename it start_offset or even start_byte_offset
@teemupiiroinenwirepas The comment is indeed misleading as the mention of 0 gives the impression that it can be 0,1, 2...
But in reality it is a byte offset.